examples/imageviewer/shared/src/androidMain/AndroidManifest.xml

```diff
@@ -1,11 +1,12 @@
 <?xml version="1.0" encoding="utf-8"?>
-<manifest xmlns:android="http://schemas.android.com/apk/res/android">
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools">
```

Comment:
These changes don't fix the crash and don't add missing FileProvider. Maybe you haven't committed the needed fix. Could you please check on your local machine?

Reply:
@mazunin-v-jb 
Thanks, i forgot to push some code.
PR updated :)